streams/src/main/java/org/apache/kafka/streams/kstream/internals/CogroupedStreamAggregateBuilder.java

```diff
@@ -201,9 +219,10 @@ private void processRepartitions(final Map<KGroupedStreamImpl<K, ?>, Aggregator<
             "-cogroup-merge",
             builder,
             CogroupedKStreamImpl.MERGE_NAME);
-        final ProcessorSupplier<K, VOut> passThrough = new PassThrough<>();
```

Comment:
I believe PassThrough is only used for the cogroup but now I think you can remove it completely.

Reply:
It's still used for some of the kstream ksteam internals so I think it's gotta stay for now

Reply:
Ok. the LGTM